extensions/markdown/README.md

```diff
@@ -0,0 +1,30 @@
+## Using Markdown-it Plugins
+
+To use a `markdown-it` plugin, goto the directory containing the `markdown-it` extension.  This will
+be found in the `extensions` folder in your vscode installation.  Install the `markdown-it` plugin
+as a node module.  For example, to install the `KaTeX` markdown-it extension you would simply type:
+
+``` sh
+npm install markdown-it-katex
+```
+
+Next, add the plugin into your [settings configuration](https://code.visualstudio.com/docs/customization/userandworkspace)
+under the `"markdown.plugs"` namespace.  This setting should contain a list for each `markdown-it` extension you wish
+to install.  The list should contain an object describing the name of the module, a list of styles that your extentsion
+may require, and options that you wish to pass to your extension.  For example, while installing the `markdown-it-katex`
+extension we would include this in your settings configuration:
+
+``` json
+    "markdown.plugins": [
+        {
+            "name": "markdown-it-katex",
+            "styles": [
```

Comment:
I'm sorry, I know that I keep saying `KaTeX` this and `KaTeX` that, and I haven't considered if you were familiar with what that is.  The KaTeX plugin is to allow math support for markdown, and I keep referencing it because it's one of the more complicated plugins you will find with respect to how styles need to be handled and that it outputs a hefty chunk of DOM to render the math for display.  So I have been using this plugin mostly as a testing grounds.  See #2273 for instance.

There is one more thing to consider.  I'm not sure how to solve this problem yet.  Some styles will want other things, like fonts or background images for instance.  Something like this:

```
@font-family
  src: url('fonts/RequireFont.ttf');
```

Ideally you would also want a place where you can keep these requirements.  Something like in the "media" folder for the markdown extension I was thinking.  I'm not sure how to tell the browser this, as I think these relative paths will point to the relative path of "webview.html"?